stubs/ExifRead/exifread/jpeg.pyi

```diff
@@ -1,5 +1,7 @@
-from _typeshed import Incomplete
+from logging import Logger
 
-logger: Incomplete
+from ._types import Reader
 
-def find_jpeg_exif(fh, data, fake_exif) -> tuple[Incomplete, Incomplete, Incomplete]: ...
+logger: Logger
+
+def find_jpeg_exif(fh: Reader, data: bytes, fake_exif: bool) -> tuple[int, bytes, bool]: ...
```

Comment:
I believe that this is just a leftover from before there was a `bool` type in Python. We had similar cases in the stdlib.

Reply:
I think that's the reason _why_ this is the case, but at least the return type should be `int`, as I don't think `bool` is ever returned. It looks like `_get_initial_base` always returns an `int` as the second item in the tuple, which means `find_jpeg_exif` always returns `int` as the third item in the tuple, even if a `bool` was passed in.

Reply:
The problem with using `bool` in the return type is that code like the following would cause a type error, although it's working fine at runtime, and is (arguably) correctly annotated:

```python
def foo(b: bool) -> ...:
    if b:
        ...
foo(find_jpeg_exif(...)[2])
```

We've found in the past that just using `bool` in cases like this works best. The only instance where is fails is if `isinstance` (or similar) is used at runtime.